make_prg/utils/misc.py

```diff
@@ -0,0 +1,10 @@
+from typing import List
+from itertools import groupby, chain
+
+
+def remove_duplicated_consecutive_elems_from_list(the_list: List) -> List:
```

Comment:
`List`s of what? Do you want [`Any`](https://docs.python.org/3/library/typing.html#typing.Any)?

Reply:
Thanks, improved the typing of this and the other function in `misc.py` (https://github.com/iqbal-lab-org/make_prg/pull/34/commits/6d7c6524d8f085538ba6db721c305eea388734d8)